network/network.proto

```diff
@@ -0,0 +1,10 @@
+package network
```

Comment:
What do you prefer ? I prefer to keep it simple and put it independant of onet, just "network" but as it does not matter much in the wire protocol, I don't have a strong opinion on this.